core/src/main/java/bio/terra/pearl/core/service/kit/LivePepperDSMClient.java

```diff
@@ -131,17 +157,17 @@ private <T> Function<String, Mono<T>> deserializeTo(Class<T> clazz) {
             }
         };
     }
-}
 
-@Component
-@Getter
-@Setter
-class PepperDSMConfig {
-    private String basePath;
-    private String secret;
-
-    public PepperDSMConfig(Environment environment) {
-        this.basePath = environment.getProperty("env.dsm.basePath");
-        this.secret = environment.getProperty("env.dsm.secret");
+    @Component
+    @Getter
+    @Setter
+    public static class PepperDSMConfig {
```

Comment:
Moving this to a `static` inner class avoids a visibility warning. This is how it should have been originally... I was just a little out of practice.